### PR TITLE
Implement reopen current tab in container shortcut

### LINKED ISSUE #8: reload current tab in different container
pretty self explanatory. would be lovely!

### PR DESCRIPTION
**Pull request by nickpwhite**
- Fixes #8 
- Opens immediately after the current tab
- Closes the current tab
- Uses `Ctrl/Cmd + Alt + #` shortcuts
- Confirmed working on both Ubuntu and MacOS

### CODE REVIEW

**Comment by nickpwhite** [comment]
@gsomoza do you need anything more from me to move this PR forward?

File: background.js [hunk 1]
```diff
@@ -34,21 +56,35 @@ async function openTabInCurrentContainer() {
 }
 
 async function openContainerTab(contextNumber) {
-  let contexts;
-  try {
-    contexts = await getContexts();
-  } catch (e) {
-    console.error(e);
+  let context = await getContextFor(contextNumber);
+  console.log(context);
```
Comment (apoorvaiyer) [inline background.js]: Should this be console.debug()? I don't see console.log being used anywhere else in this repo.

**Review by apoorvaiyer — APPROVED** [review]
First time using Github to do a review. Also, I'm not a maintainer so my comment is more on alignment with conventions already inside the repo.

Basically this is a rubber stamp.

File: background.js (hunk 1 above)
Comment (nickpwhite) [reply background.js]: I didn't actually intend to be logging there, good catch!

**Review by gsomoza — APPROVED** [review]
Great work @nickpwhite, thanks! Will merge and release.

**Comment by gsomoza** [comment]
@apoorvaiyer thanks for your review as well, awesome stuff!  👏

**Pull request merged**